flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -127,13 +128,29 @@
     private boolean forceAvro = false;
     private long autoWatermarkInterval = 200;
 
+    // ---------- statebackend related configurations ------------------------------
     /**
      * Interval in milliseconds for sending latency tracking marks from the sources to the sinks.
      */
     private long latencyTrackingInterval = MetricOptions.LATENCY_INTERVAL.defaultValue();
 
     private boolean isLatencyTrackingConfigured = false;
 
+    /** Interval in milliseconds to perform periodic materialization. */
+    private long periodicMaterializeInterval =
+            StateBackendOptions.PERIODIC_MATERIALIZATION_INTERVAL.defaultValue();
+
+    /** Interval in milliseconds for initial delay of periodic materialization. */
+    private long periodicMaterializeInitDelay =
+            StateBackendOptions.PERIODIC_MATERIALIZATION_INIT_DELAY.defaultValue();
+
+    /** Max allowed number of failures */
+    private int materializationMaxAllowedFailures =
+            StateBackendOptions.MATERIALIZATION_MAX_ALLOWED_FAILURES.defaultValue();
```

Comment:
nit: the comment says the same as the variable name so it can be removed
ditto other added fields

Reply:
Make the comment more informative.